container-cluser.tf

```diff
@@ -0,0 +1,47 @@
+resource "google_container_cluster" "primary" {
```

Comment:
Please move both tf files into the infra directory and integrate the variables. Some are already available or won't be necessary.